### DISCUSSION TITLE
Правила discussions

### DISCUSSION THREAD

**Discussion by bzick** [bzick/oh-my-backend#1] [General]
## Для чего

* Предложение по улучшению
* Обсуждение терминов и пунктов
* Просто по холиварить по технологиям.

## Правила

* не переходите на личности
* не сокорбляте
* оставайтесь профессионалом
* пишите на русском

**Comment by AlexRajvandary** [comment]
Просто хотел написать, что это очень крутой список и я как новичок благодарю за него) и у меня такой, возможно глупый вопрос: а сколько времени в целом может занять его изучение, если у меня например уже есть знания яп, но нет опыта работы?

**Reply by bzick** [reply]
Извините за задержку - много работы. 

Вопрос в том на сколько вы уже знаете что есть в списке. Но, отвечая на вопрос, на вскидку сказал бы что каждый junior вопрос можно изучить за 1-2 дня. А вот middle уже не так просто предсказать там может быть неделя. Middle+ уже не предсказать никак.

Мой личный совет к обучению будет такой . Изучите вопросы junior, запомните (не изучая) какие пункты есть у middle и middle+. Начните работать в проекте. И как только увидите что в работе "запахло" каким-то разделом/пунктами из списка — изучить middle пункты из этого раздела. В таком случае вы быстро  сможете дорасти до middle хотя бы на "своём" проекте. Дальше вам будет влиться в другие проекты уже как middle. С middle+ также, но там надо больше ковырять. 

Посыл такой: если изучать весь список до работы - не успеете поработать. Лучше изучайте список в процессе работы, с подкреплением опытом.
